locales/en-US/editor.properties

```diff
@@ -189,3 +189,8 @@ ERROR_MOVING_FILE_DIALOG_HEADER=Move Error
 UNEXPECTED_ERROR_MOVING_FILE=An unexpected error occurred when attempting to move {0} to {1}
 # {0} is the name of the file/folder being moved and {1} is the name of the folder it is being moved to
 ERROR_MOVING_FILE_SAME_NAME=A file or folder with the name {0} already exists in {1}. Consider renaming either one to continue.
+
+# extensions/extra/bramble-watch-index.html
+
+INDEX_HTML_FILE_MISSING_DIALOG_TITLE=Warning - Missing index.html file
+INDEX_HTML_FILE_MISSING_DIALOG_MESSAGE=Your project has no index.html file. Web servers look for index.html when URLs omit the filename, for example: google.com/ actually refers to google.com/index.html. Consider adding an index.html.
```

Comment:
We also should simplify the wording for this a bit since when I was trying to explain this during a session, a lot of them were confused by "web servers".